.github/policies/resourceManagement.yml

```diff
@@ -64,46 +64,53 @@ configuration:
       - isOpen
       - hasLabel:
           label: stale
+      - hasLabel:
+          label: triaged
       - not:
           hasLabel:
             label: bot-exclude
       then:
       - removeLabel:
           label: stale
-      - addLabel:
-          label: needs-triage
+      - removeLabel:
+          label: triaged
       description: Unstale Open Issues that get comments
     - if:
       - payloadType: Issue_Comment
       - not: isOpen
       - hasLabel:
           label: stale
+      - hasLabel:
+          label: triaged
       - not:
           hasLabel:
             label: bot-exclude
       then:
-      - addLabel:
-          label: needs-triage
-      description: Apply needs-triage to closed stale issues that get comments
+      - removeLabel:
+          label: triaged
+      description: Remove label triaged from closed stale issues that get comments

```

Comment:
If they're closed are we ever going to see them even untriaged? feels like this needs a positive label

Reply:
Or we need to open them maybe?

Reply:
We would need though to label almost everything as triaged, so that we see the re-opened issues.

Reply:
Btw, we do not go through closed needs-triage issues anyway, even now.

Reply:
It sounds like we should revisit the relabeling of existing issues (tracked as https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1948287):
* Existing closed issues didn't seem to be included (e.g. https://github.com/dotnet/msbuild/issues/9500)
* We should as well apply 'triaged' to all closed issues, that has 'needs-triage', but no activity occured upon closing - as this shows that 'needs-triage' just wasn't properly removed (e.g. https://github.com/dotnet/msbuild/issues/9528) 
* Above should apply as well to bugs that were relabeled as part of closing via customer (e.g. https://github.com/dotnet/msbuild/issues/9514) - we might as well revisit such a rule

That being said - only a minimum of closed issues should remain without 'triaged' label. Then the discussed action above might still be usable as is

FYI @JaynieBai

Reply:
I agree, if we apply to the closed issues "triaged", then I can go through the issues without this problem.

Reply:
@JaynieBai - it is a lower priority, but would be very nice to have. Ideally you'd first do just 'dry run' - checking the numbers of closed issues that will have triaged label and that will not have it. Than based on current numbers and projected numbers we can discuss whether to proceed.

Reply:
@JanKrivanek - why to relabel closed issues? Why do we care if Closed issue was triaged?

Reply:
@JaynieBai no need to do this.

Reply:
@rokonec Rainer pointed out that removing the triaged label from the closed issue under some circumstances (in order to return it to the triage) will not put it for triage with current queries. Unless we label closed issues as triaged.